### PR TITLE
Add /htmldeclare help.

### PR DESCRIPTION
**Pull request by Bryan-0**

### CODE REVIEW

File: commands.js [hunk 1]
```diff
@@ -1217,7 +1217,8 @@ var commands = exports.commands = {
 
 		this.add('|raw|<div class="broadcast-blue"><b>' + target + '</b></div>');
 		this.logModCommand(user.name + " declared " + target);
-	},
+	},
```
Comment (SolarisFox) [inline commands.js]: What is being changed on this line?

File: commands.js [hunk 2]
```diff
@@ -1217,7 +1217,8 @@ var commands = exports.commands = {
 
 		this.add('|raw|<div class="broadcast-blue"><b>' + target + '</b></div>');
 		this.logModCommand(user.name + " declared " + target);
-	},
+	}, 
+	htmldeclarehelp: ["/htmldeclare [message] - Anonymously announces a message with html codes. Requires: ~"],
```
Comment (Slayer95) [inline commands.js]: It would be good to state that only "safe" html codes are allowed.

**Comment by SolarisFox** [comment]
Failed the Travis build.

File: commands.js (hunk 2 above)
Comment (SolarisFox) [inline commands.js]: Do we have any documentation on what the "safe" html is?

Comment (Slayer95) [inline commands.js]: No, but you can get the list running in a PS window
`Object.keys(html4.ELEMENTS).filter(function (elem) {return html4.ELEMENTS[elem] | html4.eflags.UNSAFE})`

**Comment by Zarel** [comment]
Also "HTML codes" is pretty weird phrasing. Are you non-native in English, by any chance?

**Comment by Bryan-0** [comment]
Yes im, the english is not my first language but i think "html codes" describe it.

**Comment by kotarou3** [comment]
Firstly, HTML isn't even a language used for coding. It's "HyperText **Markup** Language"

**Comment by Slayer95** [comment]
Well, Chomsky and other linguists assert that languages are codes.

**Comment by ScottehMax** [comment]
+1 @kotarou3 

I think just "HTML" instead of "html codes" is fine.

**Comment by xfix** [comment]
(controversial opinion incoming)

HTML is a programming language. If you write `<a href="/">Link</a>`, that means **if** word "Link" is pressed **then** redirect to root page. Clearly, a programming language, just not a general purpose one.

But seriously, just say `using HTML` or something. This will stop all those discussions about what exactly HTML is.

**Comment by Zarel** [comment]
I'd argue that HTML is not a programming language. A single specific conditional does not a programming language make. Or is a light switch also a programming language?

I _would_ agree that HTML is code, though. I'd include both markup and programming languages in the "code" category.

However, "with HTML codes" is still probably grammatically incorrect. It sounds about as awkward as "with English languages". "In English" or "in HTML" are much more idiomatic ways to express that concept.

**Comment by Bryan-0** [comment]
Is that alright?

**Comment by Slayer95** [comment]
Again: `It would be good to state that only "safe" html code/tags/attributes are allowed.`

This is literally what help is good for :| Other than this, it's kind of obvious what `/htmldeclare` does.

**Comment by Bryan-0** [comment]
Updated.

**Comment by sirDonovan** [comment]
Make sure you run <code>npm test</code> before committing to check for errors. You'll also need to squash your commits into one before this can be merged.

**Comment by Bryan-0** [comment]
Ok, i squash the commits and fix the errors. @Zarel

**Pull request merged**